PocketKit/Sources/PocketKit/MyList/ItemsList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -675,24 +673,8 @@ extension SavedItemsListViewModel {
         switch syncEvent {
         case .error, .loadedArchivePage:
             break
-        case .savedItemCreated:
+        case .savedItemCreated, .savedItemsUpdated:
```

Comment:
All we need to do here now is just call `fetch` NSFetchedResultsController will handle the rest.